lib/linkedlist.js

```diff
@@ -6,8 +6,8 @@ Object.defineProperty(exports, "__esModule", {
 
 function _classCallCheck(instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError("Cannot call a class as a function"); } }
 
-var Stack = function Stack() {
-  _classCallCheck(this, Stack);
+var LinkedList = function LinkedList() {
```

Comment:
Consider putting `/lib` in your `.gitignore` since it contains Babel transpiled files.